polylang.php

```diff
@@ -36,25 +36,41 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Don't access directly.
-}
+defined( 'ABSPATH' ) || exit;
 
 if ( defined( 'POLYLANG_VERSION' ) ) {
 	// The user is attempting to activate a second plugin instance, typically Polylang and Polylang Pro.
 	require_once ABSPATH . 'wp-admin/includes/plugin.php';
 	require_once ABSPATH . 'wp-includes/pluggable.php';
+
 	if ( is_plugin_active( plugin_basename( __FILE__ ) ) ) {
 		deactivate_plugins( plugin_basename( __FILE__ ) ); // Deactivate this plugin.
 		// WP does not allow us to send a custom meaningful message, so just tell the plugin has been deactivated.
 		wp_safe_redirect( add_query_arg( 'deactivate', 'true', remove_query_arg( 'activate' ) ) );
 		exit;
 	}
-} else {
-	// Go on loading the plugin.
-	define( 'POLYLANG_VERSION', '3.8-dev' );
-	define( 'PLL_MIN_WP_VERSION', '6.2' );
-	define( 'PLL_MIN_PHP_VERSION', '7.2' );
+	return;
+}
+
+// Stopping here if we are going to deactivate the plugin (avoids breaking rewrite rules).
```

Comment:
This comment did not refer to this line. See : https://github.com/polylang/polylang/blob/3.7.3/include/class-polylang.php#L45-L48

Reply:
Moved in [7b0e18e](https://github.com/polylang/polylang/pull/1704/commits/7b0e18e60880b26f7e5e9d7df5672771b4077f92).